index.js

```diff
@@ -70,6 +70,25 @@ module.exports = function(feature, tolerance, highQuality){
       poly.coordinates.push(simpleRing);
     });
     return simpleFeature(poly, feature.properties)
+  } else if(feature.geometry.type === 'MultiPolygon') {
+    var poly = {
+      type: 'MultiPolygon',
+      coordinates: []
+    };
+    feature.geometry.coordinates.forEach(function(rings){
+      var simpleRings = [];
+      rings.forEach(function (ring) {
```

Comment:
this can be more simply expressed as a `.map` statement:

`simpleRings = rings.map( // and return simpleRing instead of pushing it onto the array`